### ISSUE TITLE
Error: 'wp-config.php' not found.

### ISSUE THREAD

**Issue by visata** [Upstatement/skela-wp-theme#65]
Hi,

I tried launching this project as documented:
`./bin/install`

It failed on the last step:

```
...
Generating optimized autoload files

Activating WordPress plugins...


Error: 'wp-config.php' not found.
Either create one manually or use `wp config create`.
```

The project looks very promising, but barely receives any updates. Are you planning on any further development?

Thanks

**Comment by mgburns** [comment]
Looks like there was an issue bringing the container up. Can you paste the output of the following command?

```
docker-compose logs wordpress
```

Successful output would look something like this:

```
wordpress_1  | [08-Dec-2020 14:54:58] NOTICE: fpm is running, pid 21
wordpress_1  | [08-Dec-2020 14:54:58] NOTICE: ready to handle connections
wordpress_1  | 2020-12-08 14:54:59,895 INFO success: php-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
wordpress_1  | 2020-12-08 14:54:59,895 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
```

We continue to use this project as the basis for our own WordPress theme development. We have a more opinionated starter theme (currently private, but that may change) that sees more frequent updates, but continue to flow updates / fixes back in to this one where appropriate.

**Comment by visata** [comment]
Thanks, it gave me the clue what needed to be fixed.

I'm developing on Windows, so I had to fix the endings of each script:
`sed -i 's/\r$//' filename
`

I had fixed **bin/** folder scripts but apparently forgot to fix **docker\scripts\setup.sh** which was responsible for wp-config.php creation. This is why the script wasn't ran.

I started building my new project on Skela but I wish it had more features inside. It would be great if you decided to share your starter theme. I'm sure you are using the proven development methods which many including me would follow.

Is that possible to access your private theme by buying some kind of license?

Thanks

**Comment by mgburns** [comment]
@visata Makes sense! We don't have many Windows users here at Upstatement, so PRs would be very appreciated. 😄 

It's likely that our batteries-included starter kit will be open sourced in the new year, but we're trying to sort out a few things internally before we do so.

Care to elaborate on what `more features inside` you would like to see?

**Comment by visata** [comment]
@mgburns That sounds great, maybe I need to wait for your batteries-included starter kit then instead of hacking the existing one :)

I'm using virtualized Ubuntu on Windows but had to make some fixes on scripts which are mac environment related, for example:

Had to change from :
``eval `sed -i '' "s/Skela/${ORIGINAL}/g" docker-compose.yml`
``
To:
``eval `sed -i -e "s/Skela/${ORIGINAL}/g" docker-compose.yml``

I haven't digged into the backend part but frontend wise I would expect the theme to contain the elements which are required for any basic website/blog to function (with some styling based on CSS methodology, framework you use), for example my first task was to create twig templates for menus (top, footer).

It would be very useful if you added more information about CI. This topic is barely covered anywhere. It's pretty difficult to sync db changes between dev, stage, production environments due to the Wordpress specifics. Usually, if you use docker during development, you would want the same container running on stage and production environments.

Correct me if I'm wrong but you don't use docker on production environment? I found this script [deploy ](https://github.com/Upstatement/skela-wp-theme/blob/main/bin/deploy) which only syncs files. How do you exactly sync databases? I assume via WP Sync DB plugin?

Thanks
